axelarnetwork/axelar-core
# feat(tss): multisig sign

## Description

- add SubmitMultisigSignatures tx
- vald call Multisig service to sign
- check timeout in abci

## Todos


## Steps to Test
Deploy contract, deposit to linked address, create pending transfer and sign. The flow should be same as before
## Expected Behaviour

## Other Notes
Will add unit test tomorrow
Haven't ran any manual test, don't know if it's working
